server/routes/courtRegister/courtMapper.ts

```diff
@@ -58,39 +58,39 @@ export function courtBuildingMapper(building: CourtBuilding): CourtBuildingDetai
   return {
     id: building.id,
     code: building.subCode,
-    name: building.buildingName,
+    name: building.buildingName as string,
```

Comment:
wrong, can be null

Reply:
I assumed he swagger is wrong, but need to look at this again, might be they can be null, checking database